src/devices/MotorHat/README.md

```diff
@@ -0,0 +1,47 @@
+﻿# MotorHat
+
+## Summary
+
+Motor HAT is an add-on board for Raspberry Pi.
+
+It consists of a pca9685 PWM driver and two motor controller chips, that together support up to 4 DC motors, or 2 Stepper motors.
+It also provides 4 extra PWM Outputs, that can be used for anything that requires PWM, (controlling a LED, a ServoMotor, etc)
+
+You can find in depth documentation on the hat in [Adafruit website](https://www.adafruit.com/product/2348)
+Or you can also get it at [Aliexpress](http://s.click.aliexpress.com/e/mTB4ZB2s)
+
+### Currently supported devices:
+- Up to 4 DC Motors
+- Up to 4 PWM Xtra channels
+- Up to 4 [ServoMotors](https://github.com/dotnet/iot/blob/master/src/devices/ServoMotor/README.md)
+
+### Not yet supported devices:
+- Up to 2 Stepper Motors
```

Comment:
there is no requirement of having utilities which is what I consider these. I'd just remove this section (unless the board explicitly mentions them)